src/Uno.UI/UI/Xaml/Controls/Control/Control.Skia.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Uno.UI.Controls;
+using Uno.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.UI.Xaml.Media;
+using Uno.Logging;
+using System.Drawing;
+using System.Linq;
+using Uno.Disposables;
+using Uno.UI;
+using View = Windows.UI.Xaml.UIElement;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class Control
+	{
+		public Control()
+		{
+			InitializeControl();
+		}
+
+		partial void UnregisterSubView()
+		{
+			//var child = this.GetChildren()?.FirstOrDefault();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)